libraries/libfc/include/fc/crypto/public_key.hpp

```diff
@@ -40,15 +40,15 @@ namespace fc { namespace crypto {
 
          // serialize to/from string
          explicit public_key(const std::string& base58str);
-         std::string to_string(const fc::yield_function_t& yield = fc::yield_function_t()) const;
+         std::string to_string(const fc::yield_function_t& yield) const;
```

Comment:
I'm not getting the point of removing this and then use `to_string({})` in so many places. What is the benefit in your opinion?

Reply:
The yield is there to prevent abuse. By defaulting the parameter, it makes it easy to miss that a yield should be provided if it is used in critical code.

Reply:
ok, it still sounds too strict but it seems that this syntax sugar was abused so I trust you.